src/test/java/br/ufg/inf/AprovDivulgInfoTest.java

```diff
@@ -0,0 +1,482 @@
+
+		PreparedStatement ps = conn.prepareStatement(user_data);
+		ps.setString(1, email);
+		ps.setString(2, senha);
+		ps.setString(3, nome);
+		ps.setString(4, cpf);
+		ps.setBinaryStream(5, null, 0);
+		ps.setString(6, recebe_divulgacao); // CAST AS recebe_divulgacao type in
+											// SQL
+		ps.setDate(7, timestamp_de_cadastramento);
+		ps.setDate(8, timestamp_de_ultima_atualizacao);
+		ps.setDate(9, timestamp_de_exclusao_logica);
+
+		try {
+			ps.executeQuery();
+		} catch (Exception e) {
+			System.out.println(e.toString());
+		}
+
+	}
+
+	@Test
+	@Ignore
+	public void testInsertAprov() throws SQLException, FileNotFoundException {
+
+		Connection conn = DriverManager.getConnection(url, usuarioDB, senhaDB);
+		Statement stm = conn.createStatement();
+
+		String user_data = "INSERT INTO aprovacao_de_divulgacao" + " (id, divulgacao_aprovada,"
+				+ " parecer_sobre_divulgacao, data_aprovacao_ou_rejeicao)" + " values" + "(?,?,?,?);";
+
+		PreparedStatement ps = conn.prepareStatement(user_data);
+		ps.setInt(1, id_aprovacao);
+		ps.setBoolean(2, aprovada);
+		ps.setString(3, parecer);
+		ps.setDate(4, data_aprovacao_ou_rejeicao);
+
+		try {
+			if (id_aprovacao < 1) {
+				System.out.println("Identificador n�o pode ser menor que 0.");
+
+			} else {
+				ps.executeQuery();
+			}
+
+		} catch (Exception e) {
+			System.out.println(e.toString());
+		}
+
+	}
+
+	@Test
+	@Ignore
+	public void testInsertCurso() throws SQLException, FileNotFoundException {
+
+		Connection conn = DriverManager.getConnection(url, usuarioDB, senhaDB);
+		Statement stm = conn.createStatement();
+
+		String user_data = "INSERT INTO curso_da_ufg" + " (nivel," + " tipo_de_resolucao, numero_da_resolucao, "
+				+ "presencial, turno)" + " values" + "(CAST(? AS nivel)," + "CAST(? AS tipo_de_resolucao),?,?,"
+				+ "CAST(? AS turno));";
+
+		PreparedStatement ps = conn.prepareStatement(user_data);
+		ps.setString(1, nivel);
+		ps.setString(2, tipo_de_resolucao);
+		ps.setInt(3, numero_da_resolucao);
+		ps.setBoolean(4, presencial);
+		ps.setString(5, turno);
+
+		try {
+			ps.executeQuery();
+		} catch (Exception e) {
+			System.out.println(e.toString());
+		}
+
+	}
+
+	@Test
+	@Ignore
+	public void testTruncate() throws SQLException {
+
+		Connection conn = DriverManager.getConnection(url, usuarioDB, senhaDB);
+		Statement stm1 = conn.createStatement();
+		Statement stm2 = conn.createStatement();
+		Statement stm3 = conn.createStatement();
+
+		String data1 = "TRUNCATE usuario;";
+		String data2 = "TRUNCATE aprovacao_de_divulgacao;";
+		String data3 = "TRUNCATE curso_da_ufg;";
+
+		try {
+			stm1.execute(data1);
+			stm2.execute(data2);
+			stm3.execute(data3);
+		} catch (Exception e) {
+			System.out.println(e.toString());
+		}
+
+	}
+
+	@Test
+	public void testBuscaUsuario()
+			throws SQLException, FileNotFoundException, NoSuchAlgorithmException, UnsupportedEncodingException {
+
+		testInsertUsuario();
```

Comment:
Isto é tido como uma má prática em geral no JUnit. Não use referencie o mesmo teste em outro teste.
Se for criar os dados, simplesmente crie eles em outro método, mas não utilize um método que esteja anotado como "@Test" dentro de outro com "@Test"